identityClient/identityClient.go

```diff
@@ -1119,3 +1119,37 @@ func (c *E3dbIdentityClient) InitiateFederationConnection(ctx context.Context, p
 	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, req, c.SigningKeys, c.ClientID, &response)
 	return response, err
 }
+
+// ConfigureFederationConnection handles a realm connecting to a federator
+func (c *E3dbIdentityClient) ConfigureFederationConnection(ctx context.Context, params ConnectFederationRequest) error {
+	path := c.Host + identityServiceBasePath + "/" + federationResourceName + "/connection/" + params.ConnectionID.String()
+	req, err := e3dbClients.CreateRequest(http.MethodPost, path, params)
+	if err != nil {
+		return err
+	}
+	return e3dbClients.MakeSignedServiceCall(ctx, c.requester, req, c.SigningKeys, c.ClientID, nil)
+}
+
+// CompleteFederationConnection handles the activation of a federation request
+func (c *E3dbIdentityClient) CompleteFederationConnection(ctx context.Context, params ConnectFederationSaveRequest, credentials map[string]string) error {
+	// Change Host to Primary Realm Endpoint
+	path := params.PrimaryRealmEndpoint + identityServiceBasePath + "/" + federationResourceName + "/activate/" + params.ConnectionID.String()
+	req, err := e3dbClients.CreateRequest(http.MethodPut, path, params)
+	if err != nil {
+		return err
+	}
+	// Set Credential Token
+	for key, val := range credentials {
+		var isAuthHeader bool
+		for _, authHeader := range AuthenticationHeaders {
+			if key == authHeader {
+				isAuthHeader = true
+				break
+			}
+		}
+		if isAuthHeader {
+			req.Header.Add(key, val)
+		}
+	}
+	return e3dbClients.MakeSignedServiceCall(ctx, c.requester, req, c.SigningKeys, c.ClientID, nil)
```

Comment:
not asking you to do this, but just to be aware that "ideally" we would make a new auth method that does the logic in Ines 1141 - 1152 and use that versus the signed service call handler.